### ISSUE TITLE
Bucket tests will fail unless you specify helper path

### ISSUE THREAD

**Issue by Ash258** [lukesampson/scoop#3468]
For bucket tests helpers are not needed at all.

Since f63219fd1d59460e8416bdbf9575b3ec47959c85 all external buckets test will fail unless you add environment variable for scoop_helpers. 

https://ci.appveyor.com/project/Ash258/scoop-ash258/builds/24502846/job/66ps0e8csga27tj5

There should be check if that environment variable exists and if no, just use hardcoded path instead of forcing users to duplicate redundant three lines into own pipeline configuration (As they really do not need them).

**Comment by r15ch13** [comment]
Like so? `test\bin\init.ps1`
```powershell
if ($env:CI_WINDOWS -eq $true) {
    $helpersdir = $env:SCOOP_HELPERS, "C:\projects\helpers" | Where-Object { -not [String]::IsNullOrEmpty($_) } | Select-Object -First 1

    if(!(Test-Path $helpersdir)) {
        New-Item -ItemType Directory -Path $helpersdir
    }
    if(!(Test-Path "$helpersdir\lessmsi\lessmsi.exe")) {
        Start-FileDownload 'https://github.com/activescott/lessmsi/releases/download/v1.6.3/lessmsi-v1.6.3.zip' -FileName "$helpersdir\lessmsi.zip"
        & 7z.exe x "$helpersdir\lessmsi.zip" -o"$helpersdir\lessmsi" -y
    }
    if(!(Test-Path "$helpersdir\innounp\innounp.exe")) {
        Start-FileDownload 'https://raw.githubusercontent.com/ScoopInstaller/Binary/master/innounp/innounp048.rar' -FileName "$helpersdir\innounp.rar"
        & 7z.exe x "$helpersdir\innounp.rar" -o"$helpersdir\innounp" -y
    }
}
```

**Comment by Ash258** [comment]
I guess this should work.

I am just tired of adding these lines into all buckets

https://github.com/Ash258/scoop-Ash258/commit/788260fae60651cc97f9ed09a60000634701432a
https://github.com/Ash258/Scoop-JetBrains/commit/659eb1f1d3ffe336588e62b2282620ca4b8794d5
https://github.com/Ash258/GenericBucket/commit/1defd6dfda9752996aae5958ff7cbcffbbb50972

I am too lazy to add it into sysinternals bucket too :D

**Comment by Ash258** [comment]
Fixed in f52b339523a7ef4884407eaeb1fe6a22d7c2bec8
